content/docs/deployment/csminstallationwizard/src/templates/operator/csm-isilon-1.9.0.template

```diff
@@ -0,0 +1,470 @@
+apiVersion: storage.dell.com/v1
+kind: ContainerStorageModule
+metadata:
+  name: isilon
+  namespace: $NAMESPACE
+spec:
+  driver:
+    csiDriverType: "isilon"
+    csiDriverSpec:
+      # fsGroupPolicy: Defines if the underlying volume supports changing ownership and permission of the volume before being mounted.
+      # Allowed values: ReadWriteOnceWithFSType, File , None
+      # Default value: ReadWriteOnceWithFSType
+      fSGroupPolicy: "$FSGROUP_POLICY"
+      # storageCapacity: Helps the scheduler to schedule the pod on a node satisfying the topology constraints, only if the requested capacity is available on the storage array
+      # Allowed values:
+      #   true: enable storage capacity tracking
+      #   false: disable storage capacity tracking
+      storageCapacity: $STORAGE_CAPACITY_ENABLED
+    # Config version for CSI PowerScale v2.8.0 driver
+    configVersion: v2.8.0
+    authSecret: isilon-creds
+    replicas: $CONTROLLER_COUNT
+    dnsPolicy: ClusterFirstWithHostNet
+    # Uninstall CSI Driver and/or modules when CR is deleted
+    forceRemoveDriver: true
+    common:
+      # Image for CSI PowerScale driver v2.8.0
+      image: "dellemc/csi-isilon:v2.8.0"
```

Comment:
Update all the driver version to v2.9.0 for xx1.9.0 templates

Reply:
I agree with @chimanjain.  Please make updates for platforms here